goosebit.yaml

```
@@ -1,32 +1,46 @@
 ## Settings to adjust for each installation
+## Settings are listed with single #, comments with 2 #
 
-# Port to host the server on, default:
+## Port to host the server on, default:
 # port: 60053 # GOOSE ;)
```

I suggest this convention as an alternative:
```suggestion
#port: 60053 # GOOSE ;)
```